final_project/flaskr/schema.sql

```diff
@@ -54,13 +43,13 @@ database using its coordinates
 a user can vote only once per bikerack.  */
 
 CREATE TABLE votes (
-    vote_timestamp INTEGER,
-    vote_type TEXT,
+    user_id INTEGER,
```

Comment:
This table needs  a primary key of some kind, no? Either `(user_id, rack_id)` or maybe a `vote_id` column that is just an `INTEGER AUTOINCREMENT`?

Reply:
I'll go for vote_id

Reply:
Sounds good. You'll want to update this table based on user_id, so probably also add an index on that.